.github/workflows/automationTemplate.yaml

```diff
@@ -0,0 +1,146 @@
+name: Oryx-SDK-Runtime-Automation-Template
+
+on:
+  workflow_call:
+    inputs:
+      releaseDate:
+        required: false
+        type: string
+      platformName:
+        required: true
+        type: string
+      platformNameCamelCase:
+        required: true
+        type: string
+
+  workflow_dispatch:
+    inputs:
+      releaseDate:
+        description: 'The date to check for new releases. Format: yyyy-mm-dd'
+        required: false
+        type: string
+      platformName:
```

Comment:
Could you please clarify why we need these two names (platformName and platformNameCamelCase) to be different? Could we generate one from the other (for example just making platformNameCamelCase to lower case to create platformName)?
If we actually need these names to be very specific (for example because we'll go look for a URL with these names or something), please consider adding that to the comments to make it clearer.

Reply:
We need a camel case name since for example: We need to map `dotnet` to `DotNetCore` since our Pipelines have camel casing. However, I was able to remove the extra input by [adding this as an environment variable](https://github.com/microsoft/Oryx/pull/1766/commits/0a5663ef08be8fd331346efb40914d734f71657d).